snmalloc-sys/src/lib.rs

```diff
@@ -57,14 +57,14 @@ extern "C" {
     /// If `p` is null, it behaves as [`sn_malloc`]. If `newsize` is larger than
     /// the original `size` allocated for `p`, the bytes after `size` are
     /// uninitialized.
-    pub fn sn_realloc(p: *mut c_void, newsize: usize) -> *mut c_void;
+    pub fn realloc(p: *mut c_void, newsize: usize) -> *mut c_void;
 
     /// Free previously allocated memory.
     /// The pointer `p` must have been allocated before (or be null).
-    pub fn sn_free(p: *mut c_void);
+    pub fn free(p: *mut c_void);
 
     /// Return the available bytes in a memory block.
-    pub fn sn_malloc_usable_size(p: *const c_void) -> usize;
+    pub fn sn_rust_usable_size(p: *const c_void) -> usize;
```

Comment:
Do you still want the `sn_` prefix?

Reply:
Let s say I m not bothered by it. Might give a clearer path of its origin tough.